pkg/asset/installconfig/ibmcloud/client.go

```diff
@@ -140,8 +123,51 @@ func (c *Client) GetCISInstance(ctx context.Context, crnstr string) (*resourceco
 	return resourceInstance, nil
 }
 
+// GetDNSRecordsByName gets DNS records in specific Cloud Internet Services instance
+// by its CRN, zone ID, and DNS record name.
+func (c *Client) GetDNSRecordsByName(ctx context.Context, crnstr string, zoneID string, recordName string) ([]dnsrecordsv1.DnsrecordDetails, error) {
+	// Set CIS DNS record service
+	dnsService, err := dnsrecordsv1.NewDnsRecordsV1(&dnsrecordsv1.DnsRecordsV1Options{
+		Authenticator:  c.Authenticator,
+		Crn:            &crnstr,
+		ZoneIdentifier: &zoneID,
+	})
+	if err != nil {
+		return nil, err
+	}
+
+	// Get CIS DNS records by name
+	records, _, err := dnsService.ListAllDnsRecordsWithContext(ctx, &dnsrecordsv1.ListAllDnsRecordsOptions{
+		Name: &recordName,
+	})
+	if err != nil {
+		return nil, errors.Wrap(err, "could not retrieve DNS records")
+	}
+
+	return records.Result, nil
+}
+
+// GetDNSZoneIDByName gets the CIS zone ID from its domain name.
+func (c *Client) GetDNSZoneIDByName(ctx context.Context, name string) (string, error) {
+	_, cancel := context.WithTimeout(ctx, 1*time.Minute)
+	defer cancel()
```

Comment:
```suggestion
```

Don't need this. I know this is coming from other functions in here, but a recent comment from a different PR review pointed this out.